code/stable/pdcontroller/src/python/Constants.py

```diff
@@ -5,5 +5,5 @@
 
 ## \brief Structure for holding the constant values
 class Constants:
-    AbsTol = 1.0e-10
-    RelTol = 1.0e-10
+    ABSTOL = 1.0e-10
+    RELTOL = 1.0e-10
```

Comment:
What I'd _really_ expect here is `ABS_TOL` and `REL_TOL`, to preserve the separation between the two parts of the variable name. Is there a reasonably straightforward way of doing this? (If not, this is likely getting "into the weeds"; NoPCM explicitly uses underscores for its variable names, so we may just want to do that for Projectile as well, potentially enforcing variable names to be all lowercase?)

Reply:
That's a good point!  It should be possible to do better.  Off the top of my head, I think if we just insert a `_` wherever there's a lowercase letter on the left and an uppercase letter on the right, it would match what we expect in just about every case.  I'll see what functions are available, and get something working.

Reply:
I made new function to do the renaming, and it handled the few examples we have well.